airbyte-workers/src/main/java/io/airbyte/workers/config/ContainerOrchestratorConfigBeanFactory.java

```diff
@@ -87,13 +105,27 @@ public ContainerOrchestratorConfig kubernetesContainerOrchestratorConfig(
       environmentVariables.put(DD_VERSION_ENV_VAR, System.getenv(DD_VERSION_ENV_VAR));
     }
 
+    // Environment variables for ApiClientBeanFactory
+    environmentVariables.put(CONTROL_PLANE_AUTH_ENDPOINT_ENV_VAR, controlPlaneAuthEndpoint);
+    environmentVariables.put(DATA_PLANE_SERVICE_ACCOUNT_CREDENTIALS_PATH_ENV_VAR, dataPlaneServiceAccountCredentialsPath);
+    environmentVariables.put(DATA_PLANE_SERVICE_ACCOUNT_EMAIL_ENV_VAR, dataPlaneServiceAccountEmail);
+    environmentVariables.put(AIRBYTE_API_AUTH_HEADER_NAME_ENV_VAR, containerOrchestratorApiAuthHeaderName);
+    environmentVariables.put(AIRBYTE_API_AUTH_HEADER_VALUE_ENV_VAR, containerOrchestratorApiAuthHeaderValue);
+    environmentVariables.put(INTERNAL_API_HOST_ENV_VAR, containerOrchestratorApiHost);
```

Comment:
Since these are part of the EnvConfigs, I tried adding those to the envMap in OrchestratorConstants, but while they showed up in the pod's envMap.json when exec'ing into the orchestrator, those env vars didn't seem to get set on the orchestrator itself. This approach of adding them to this environmentVariables map did set them correctly.

Reply:
Huh, I wonder what the purpose of that envMap.json file is if it doesn't actually set the env vars? Just curious if you looked into it at all, not a blocker for this PR